### PR TITLE
Add documentation for `realtimeSubscribeToAttendeeIdPresence`callback…

### PR DESCRIPTION
**Pull request by devalevenkatesh**
… parameters

**Issue #:**
#1498 

**Description of changes:**
- Add documentation for `realtimeSubscribeToAttendeeIdPresence`callback parameters.
- Created a new type `RealtimeSubscribeToAttendeeIdPresenceCallback` to provide correct documentation based on params in typedoc.
- Tested the existing JS SDK demo without making any changes to the handlers type to see whether we run into any error, no problem found.
- Added `strictFunctionTypes ` as `true` to base tsconfig used in the JS SDK demo as well and still the build ran fine and the demo worked as expected. We do not use that check in our tsconfig.json just added for test purposes.
- Hence, the new type addition and changing the callback `realtimeSubscribeToAttendeeIdPresence` type did not create breaking change problems.

**Testing**

1. Have you successfully run `npm run build:release` locally? Yes.
2. How did you test these changes? 
- By running the `npm run doc` and then checked the rendered typedoc HTML page by opening docs/interfaces/realtimecontroller.html.
- This is just the API documentation update.
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? NA.
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? No, but added the documenation.
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? NA.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/realtimecontroller/RealtimeController.ts [hunk 1]
```diff
@@ -59,7 +59,35 @@ export default interface RealtimeController {
 
   /**
    * Subscribes to changes in attendee ids in order to discover attendee ids to
-   * subscribe and unsubscribe to for volume indicator updates.
+   * subscribe and unsubscribe to for volume indicator updates. The subscribed callback is called with below parameters:
+   *
+   * `attendeeId`:
+   * Unique `AttendeeId` received from `CreateAttendee` API representing an attendee.
```
Comment (ltrung) [inline src/realtimecontroller/RealtimeController.ts:65]: Should we say internal Chime attendee ID created by `CreateAttendee` API?

File: src/realtimecontroller/RealtimeController.ts [hunk 2]
```diff
@@ -59,7 +59,35 @@ export default interface RealtimeController {
...
+   * It is also provided to differentiate the scenarios between normal attendee leave and the attendee dropping due to re-connection issues.
+   *
+   * In re-connection scenarios, if an attendee drops and could never join back successfully, the JS SDK will call this callback setting
+   * the `dropped` value to a boolean value received from Amazon Chime backend, and it will set the `present` parameter to `false`.
+   *
+   * `posInFrame`: {@link RealtimeAttendeePositionInFrame}
+   *
+   * This object indicates which attendee out of how many total attendees the update is for.
+   * For example, if you were to join on a call with 3 total attendees, you would get presence callbacks for attendeeIndex 0, attendeeIndex 1, attendeeIndex 2 out of the total attendeesInFrame of 3.
+   * The first presence update you receive is for all of the present attendees and later presence updates are for individual attendees as they join and leave.
```
Comment (ltrung) [inline src/realtimecontroller/RealtimeController.ts:90]: >  The first presence update you receive is for all of the present attendees

Should we make it clear that the callback is trigger for each attendee once you join a meeting? This sounds like all of them at once.

File: src/realtimecontroller/RealtimeController.ts (hunk 1 above)
Comment (devalevenkatesh) [reply src/realtimecontroller/RealtimeController.ts]: Yup updated it. Thanks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+- Add API documentation for `realtimeSubscribeToAttendeeIdPresence` callback parameters.
```
Comment (ltrung) [inline CHANGELOG.md:11]: nit: Should we mention that you add the type `RealtimeSubscribeToAttendeeIdPresenceCallback`?

Comment (devalevenkatesh) [reply CHANGELOG.md]: Thanks for the catch! Thats right. Updated it.

**Review by ltrung — APPROVED** [review]

**Pull request merged**